tla/ccfraft.tla

```diff
@@ -481,6 +487,19 @@ AppendEntriesBatchsize(i, j) ==
      \* This can be redefined to send bigger batches of entries.
     {nextIndex[i][j]}
 
+
+PlausibleSucessorNode(i) ==
+    \* Find a plausible successor node for i
+    \* Note that while CHOOSE is deterministic, it may not pick the same
+    \* node as the implementation, which iterates through an unordered_map
+    \* when performing its selection with the same criteria.
+    LET
+        activeServers == Servers \intersect removedFromConfiguration
```

Comment:
I think `\intersect` should be `\` (set difference). `Servers \intersect removedFromConfiguration = removedFromConfiguration` since `removedFromConfiguration \subseteq Servers` according to the `RemovedFromConfigurationTypeInv`

Reply:
Yes, absolutely, my mistake.